Makefile.am

```diff
@@ -195,6 +195,7 @@ LCOV_FILTER_PATTERN = \
 	-p "src/crypto/ctaes" \
 	-p "src/minisketch" \
 	-p "src/secp256k1" \
+	-p "src/simplicity/secp256k1" \
```

Comment:
In 50916cf1081d842865611130cb51dda38163cc95:

Probably we didn't want to filter this. (I'm kinda surprised that the upstream libsecp is filtered. But I guess they have their own coverage analysis.)

Reply:
While we're at it, we should stick /nix/store into this list and src/include/boost. I'm not sure why we need to do this and Bitcoin doesn't (well, the answer is "NixOS") but it's harmless and helpful for Russell and me.

Reply:
I'm a little hesitant to add nix-specific fixes here, and rather just patch the file in https://github.com/roconnor-blockstream/elements-nix/

Reply:
I've removed `src/simplicity/secp256k1` from the filter as part of #1390.